Подушки для сидения

Sosiskoed opened the pull request:
Подушки для сидения. Как табуретки, только подушки. Мягенькие.
Пока без спрайтов.

Sosiskoed committed:
Подушки для сидения

Подушки для сидения. Как табуретки, только подушки. Мягенькие.
Пока без спрайтов.

fentress8975 commented:
Я конечно дико извиняюсь, что так резко, но
Они будут вместо табуреток или... Можно будет устроить ими битву?
(тот и тот вариант хорош, я знаю)

Sosiskoed commented:
Они будут вместе с табуретками, но можно устроить ими и битву, так как они берутся в руки. Крафта пока нет, равно как и спрайтов в руке. Позже добавлю спрайты на полу.

fentress8975 commented:
Можно ещё сделать их брутальнее и добавить возможность задушить подушкой
Хотя по идее подушка не должна наносить урона(если только не поменять наполнение)

Sosiskoed committed:
Сосисочное обновление 20.03.2018

Крафт подушек из металла;
Небольшой фикс в названии подушки;
Добавление спрайта для подушки;
Переработка бара, кухни и гидропоники.

Sosiskoed committed:
Небольшие правки

Добавил интеркомы на стены, добавил дверь из бара к кухне/гидропонике.

Sosiskoed committed:
Коллекторы

Вспомнил и перенес коллекторы за стекло. Теперь они слева от сингулярности.
